### ISSUE TITLE
best model

### ISSUE THREAD

**Issue by Jung-Echo** [G-U-N/PyCIL#56]
Could you help me locate the code that retrieves the best model during each training session? I had a look, but I couldn't find any code that identifies the best performing model throughout the training process. I suspect that the model.eval_task() line in the trainer only evaluates the model saved after the last epoch, am I correct?

**Comment by zhoudw-zdw** [comment]
I am not sure about your question. What do you mean by 'the best model during each training session'?

Since we do not have a validation set, we train for a specific number of epochs for each incremental task. On the other hand, if you intend to save and re-evaluate the model, perhaps you should refer to this page:

https://pytorch.org/tutorials/beginner/saving_loading_models.html
